### PR TITLE
rutracker - always use https

### PR DESCRIPTION
**Pull request by Danfocus**
### Motivation for changes:

Sometimes rutracker.org available only throw https

### Detailed changes:

- Add new property use_ssl(boolean)
- Small code changes

### Addressed issues:

- None

### Config usage if relevant (new plugin or updated schema):
```
    rutracker_auth:
      username: 'username_here'
      password: 'password_here'
      use_ssl: yes
```
### Log and/or tests output (preferably both):

- None

### CODE REVIEW

**Comment by liiight** [comment]
Why make it optional? Why not always use https?

**Comment by Danfocus** [comment]
> Why make it optional? Why not always use https? 
I check now - rutracker always redirect to https.
Changes are made

File: flexget/plugins/sites/rutracker.py [hunk 1]
```diff
@@ -59,13 +61,14 @@ class RutrackerAccount(Base):
 
 class RutrackerAuth(AuthBase):
     """Supports downloading of torrents from 'rutracker' tracker
-       if you pass cookies (CookieJar) to constructor then authentication will be bypassed and cookies will be just set
```
Comment (cvium) [inline flexget/plugins/sites/rutracker.py]: This line doesn't look longer than 120 chars, so why the linebreak?

File: flexget/plugins/sites/rutracker.py [hunk 2]
```diff
@@ -137,7 +142,8 @@ def on_task_urlrewrite(self, task, config):
         else:
             auth_handler = self.auth_cache[username]
         for entry in task.accepted:
-            if entry['url'].startswith('http://rutracker.org/forum/viewtopic.php'):
+            if entry['url'].startswith(
+                    '{}/forum/viewtopic.php'.format(BASE_URL)):
```
Comment (cvium) [inline flexget/plugins/sites/rutracker.py]: Same here. Another weird linebreak.

File: flexget/plugins/sites/rutracker.py [hunk 3]
```diff
@@ -95,15 +98,17 @@ def __init__(self, login, password, cookies=None, db_session=None):
     def __call__(self, r):
         url = r.url
         id = re.findall(r'\d+', url)[0]
```
Comment (cvium) [inline flexget/plugins/sites/rutracker.py]: While you're already in there, you should change this variable name to something other than `id`.

File: flexget/plugins/sites/rutracker.py (hunk 1 above)
Comment (Danfocus) [inline flexget/plugins/sites/rutracker.py]: revert it

File: flexget/plugins/sites/rutracker.py (hunk 2 above)
Comment (Danfocus) [inline flexget/plugins/sites/rutracker.py]: remove linebreak

File: flexget/plugins/sites/rutracker.py (hunk 3 above)
Comment (Danfocus) [inline flexget/plugins/sites/rutracker.py]: change to t_id (thread_id)

**Comment by Danfocus** [comment]
Changes are made

**Pull request merged**